### PR TITLE
Cloud SIEM content release notes - July 3, 2024

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds Cloud SIEM content release notes.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

Per [request in Slack](https://sumologic.slack.com/archives/C03ASV6SJV8/p1720019993174979).

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, could you review this PR?

File: blog-cse/2024-07-03-content.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* [New] JFrog Artifactory - Login Access logs
+* [New] JFrog Artifactory - Request Logs
+* [New] Synergis Genetec - all
+* [Updated] AWS EKS - Custom Parser
+   * Keys updated: `'srcDevice_ip'`, `'http_response_statusCode'`, `'http_url'`, `'http_userAgent'`, `'user_username'`, `'user_userId'`, `'action'`, `'device_k8s_namespace'`
+* [Updated] Abnormal Security Threats
+   * Keys updated: `'threat_referenceUrl'`, `'email_subject'`, `'resource'`, `'email_sender'`, `'user_email'`, `'user_username'`, `'targetUser_email'`, `'action'`, `'threat_identifier'`, `'user_authDomain'`, `'srcDevice_ip'`, `'email_messageId'`, `'srcDevice_hostname'`, `'threat_name'`, `'threat_category'`, `'timestamp'`
+* [Updated] Cisco ASA 305011-12 JSON
+   * Keys updated: `'user_authDomain'`, `'user_username'`
+* [Updated] Github JSON
```
Comment (kimsauce) [inline blog-cse/2024-07-03-content.md:38]:
```suggestion
* [Updated] GitHub JSON
```

**Review by kimsauce — APPROVED** [review]
one fix!

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**